charts/rancher-backup-crd/rancher-backup-crd/1.0.500-rc1+up1.0.5-rc1/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-resources-system
+  catalog.cattle.io/release-name: rancher-backup-crd
+apiVersion: v2
+appVersion: 1.0.5-rc1
```

Comment:
This should be 1.1.1 now, based on discussion

Reply:
fixed.